### DISCUSSION TITLE
When I upload folders using webdirectory I dont get the webkitRelativePath in the file name. Can anyone help.

### DISCUSSION THREAD

**Discussion by puneety95** [rpldy/react-uploady#609] [General]
This is needed so as to maintain the folder structure at the server when I save the file. This way all the subfolders and directories will be created.

**Comment by yoavniran** [comment]
hey @puneety95 
I believe the [answer](https://github.com/rpldy/react-uploady/discussions/521#discussioncomment-5968310) is related to this question as well.
Let me know if you have more questions or issues
